build-logic/basics/src/main/kotlin/gradlebuild/basics/BuildParams.kt

```diff
@@ -110,6 +113,9 @@ object BuildParams {
     const val TEST_SPLIT_ONLY_TEST_GRADLE_VERSION = "onlyTestGradleVersion"
     const val TEST_JAVA_VENDOR = "testJavaVendor"
     const val TEST_JAVA_VERSION = "testJavaVersion"
+    const val AUTO_DOWNLOAD_ANDROID_STUDIO = "autoDownloadAndroidStudio"
+    const val RUN_ANDROID_STUDIO_IN_HEADLESS_MODE = "runAndroidStudioInHeadlessMode"
+    const val STUDIO_HOME = "studio.home"
```

Comment:
```suggestion
    const val STUDIO_HOME = "studioHome"
```

Can we use `studioHome` here, so the parameter is camel case like the other ones?

Reply:
Of course, fixed